Resources/Public/JavaScript/Script.js

```diff
@@ -0,0 +1,11 @@
+(function() {
+    $( document ).ready(function() {
```

Comment:
Do we really need jquery here? Would prefer not to...

Reply:
*Updated to JavaScript*